07-ipcc.Rmd

```diff
@@ -1,11 +1,157 @@
+---
+output:
+  pdf_document:
+    toc: true
+    bibliography: book.bib
```

Comment:
please delete table of content and references. this will automatically be rendered.